www/shared/styles/imports/variables.css

```diff
@@ -0,0 +1,69 @@
+@import "./custom-medias";
+
+/* ==========================================================================
+   Colors
+   ========================================================================== */
+
+:root {
+    --color-white: #fff;
+    --color-black: #000;
+}
+
+/* ==========================================================================
+   Typography
+   ========================================================================== */
+
+:root {
+    --font-montserrat: "Montserrat";
+    --font-family: var(--font-montserrat), sans-serif;
+
+    --font-weight-regular: 400;
+    --font-weight-bold: 700;
+}
+
+/* ==========================================================================
+   Z-index
+   ========================================================================== */
+
+:root {
+    --z-index-base: 0;
+    --z-index-header: 2;
+}
+
+/* ==========================================================================
+   Transition / Animations timing
+   ========================================================================== */
+
+:root {
+    --duration-fast: 0.15s;
+    --duration-standard: 0.3s;
+    --duration-long: 0.4s;
+}
+
+/* ==========================================================================
+   Layout vars
+   ========================================================================== */
+
+:root {
+    /* App maximum width - if this is changed you need to update the custom media --layout-lte-container-width
+    ============================================= */
+    --layout-container-width: 140rem;
+
+    /* Spacing around main contents of the website - if this is changed you need to update the custom media --layout-lte-container-width
+    ============================================= */
+    --layout-edge-spacing-x: 5%;
+
+    /* Header
+    ============================================= */
```

Comment:
I'm not a fan of these dividers. They actually confuse me because it looks like it's separating a previous section from this (and the description is actually above the divider) 😕

Reply:
This follows the `idiomatic-css` principles:

https://github.com/necolas/idiomatic-css#3-comments